### DISCUSSION TITLE
Sharing auth between two nextjs application using multi-zones.

### DISCUSSION THREAD

**Discussion by bytesagar** [vercel/next.js#46147] [Help]
Hello I have two applications, one is hosted at xyz.co and another site at xyz.co/docs using next multi zones. I have a client side auth implemented in xyz.co. I want to use same authentication for the ```/docs``` . I am storing my jwt tokens in cookies.
Is there anyway I can use same authentication in both sites?

**Comment by roymhmok** [comment]
Add `sameSite: "lax"` to your cookie options

**Comment by alamenai** [comment]
I started learning the multi-zones structures and started asking myself this question.

@roymhmok , please could you add more details?
